google-cloud-datastore/src/test/java/com/google/cloud/datastore/ITDatastoreTest.java

```diff
@@ -1108,30 +1104,36 @@ public void testRetryableException() throws Exception {
     EasyMock.verify(rpcFactoryMock, rpcMock);
   }
 
-  @Test
-  public void testNonRetryableException() throws Exception {
+  @Test(expected = DatastoreException.class)
+  public void testNonRetryableException() {
     LookupRequest requestPb = LookupRequest.newBuilder().addKeys(KEY1.toPb()).build();
     EasyMock.expect(rpcMock.lookup(requestPb))
         .andThrow(
             new DatastoreException(DatastoreException.UNKNOWN_CODE, "denied", "PERMISSION_DENIED"))
         .times(1);
     EasyMock.replay(rpcFactoryMock, rpcMock);
-    Datastore datastore = rpcMockOptions.getService();
-    thrown.expect(DatastoreException.class);
-    thrown.expectMessage("denied");
+    Datastore datastore = null;
+    try {
+      datastore = rpcMockOptions.getService();
+    } catch (DatastoreException e) {
+      assertEquals("denied", e.getMessage());
+    }
     datastore.get(KEY1);
     EasyMock.verify(rpcFactoryMock, rpcMock);
   }
```

Comment:
not sure if `expected = DatastoreException.class` is needed here and in the test below.
Does `EasyMock.verify(rpcFactoryMock, rpcMock)` throws `DatastoreException`?